sections/featured-collection.liquid

```diff
@@ -43,7 +43,7 @@
     {% endunless %}
   
     <slider-component class="slider-mobile-gutter">
-      <ul id="Slider-{{ section.id }}" class="grid grid--2-col product-grid contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %}{% if section.settings.columns_desktop == 1 %} grid--full-width{% else %} grid--{{ section.settings.columns_desktop }}-col-desktop{% endif %}{% if products_to_display == 4 or section.settings.collection == blank %} grid--2-col-tablet{% else %} grid--3-col-tablet{% endif %}{% if section.settings.collection == blank %} grid--4-col-desktop{% endif %}{% if section.settings.collection.all_products_count > 2 and section.settings.swipe_on_mobile and section.settings.products_to_show > 2 %} slider slider--tablet grid--peek{% endif %}" role="list">
+      <ul id="Slider-{{ section.id }}" class="grid product-grid contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %}{% if section.settings.columns_desktop == 1 %} grid--full-width{% else %} grid--{{ section.settings.columns_desktop }}-col-desktop{% endif %}{% if section.settings.collection == blank %} grid--2-col-tablet-down{% else %} grid--{{ section.settings.columns_mobile }}-col-tablet-down{% endif %}{% if section.settings.collection == blank %} grid--4-col-desktop{% endif %}{% if section.settings.collection.all_products_count > 2 and section.settings.swipe_on_mobile and section.settings.products_to_show > 2 %} slider slider--tablet grid--peek{% endif %}" role="list">
```

Comment:
Ugh, this whole class definition.. so unreadable. (not a new problem)

Reply:
sweetttt. I know we like to avoid moving these to assigns but some of these are too ridiculous. If I can't parse exactly what's going on in a few seconds, it's not good.